model-optimizer/extensions/back/compress_quantized_weights.py

```diff
@@ -212,5 +224,12 @@ def replace_pattern(self, graph: Graph, match: Dict[str, Node]):
         if np.issubdtype(dst_type, np.floating):
             dst_type = data_type_str_to_np(graph.graph['cmd_params'].data_type)
 
-        self.quantize_data(fake_quantize, dst_type)
-        self.dequantize_data(fake_quantize, dst_type)
+        levels = fake_quantize.levels
+        quantized_type, mode = None, None
+        for a in sorted(self.QUANTIZATION_MAP):
```

Comment:
nit: rename variable `a` to something more meaningful

Reply:
Done